src/systemctl/systemctl-edit.c

```diff
@@ -342,6 +343,10 @@ int verb_edit(int argc, char *argv[], void *userdata) {
         if (strv_isempty(names))
                 return log_error_errno(SYNTHETIC_ERRNO(ENOENT), "No units matched the specified patterns.");
 
+        if (arg_stdin && strv_length(names) != 1)
```

Comment:
Hmm, I think editing drop-ins of multiple units with the same content is a valid thing to do? I don't really like this limitation.

Say you want to disable CPU accounting for a.service and b.service:

```
systemctl edit --drop-in=disable-cpu-accounting.conf --stdin a.service b.service <<EOF
[Service]
CPUAccounting=no
EOF
```

Seems perfectly legit and useful?

Reply:
Updated like that.